ang/civicase/CaseDetails--tabs--summary.html

```diff
@@ -44,6 +44,78 @@
     </div>
   </div>
   <!-- End - Subject and QuickLinks -->
+  <div class="civicase__summary-tab-row clearfix">
+    <div class="civicase__summary-tab-tile col-lg-6">
+      <!-- next milestone card -->
+      <h3> Next Milestone </h3>
```

Comment:
This is actually the header of the panel. So `panel-header` needs to be used inside ` <div class="panel panel-default">` and should be styled accordingly.

Reply:
According to me adding it as a panel header and writing extra override styles to it is just a overkill. We just need margin to this element but if we apply panel-header we need to reset many properties that we don't need anyways . Increasing the complexity and weight of the code. 
Please let me know if we need a call for it.